adding /desks to back end

rebecalsantos opened the pull request:
updating recreate_schema.sql and populate_db.sql to match benjamin changes for desk positions.
Updating \api\desks to add the new colums

rebecalsantos committed:
adding /desks to back end

updating recreate_schema.sql and populate_db.sql to match benjamin changes for desk positions.
Updating \api\desks to add the new colums

Benjamin-GitHub commented on server/populate_db.sql:
```diff
@@ -5,12 +5,15 @@ INSERT INTO desk_user (username) VALUES ('Adeola');
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 4',175,170,180);
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 5',0,340,0);
+INSERT INTO desk (desk_name,x,y,r) VALUES ('desk 6',175,340,180);
+];
```
This data are correct. Thanks Rebeca!

Benjamin-GitHub commented on server/api.js:
```diff
@@ -16,8 +16,25 @@ router.get("/", (req, res) => {
+				id: row.id,
+				desk: row.desk_name,
+				xCoor:row.x,
+				yCoor:row.y,
```
Please check the rule of naming here "xCoor" & "yCoor". It seems you used different rule in line 50 => "desk_id " !?

rebecalsantos replied:
this is already fixed in another pr. Many thanks for reviewing

Benjamin-GitHub reviewed (commented):
Nice job Rebeca!

rebecalsantos committed:
add desk to server

added coordinates to desks in populate_db.sql
added the missing fields to apizdesks
